XamarinCommunityToolkit/Views/CameraView/CameraView.shared.cs

```diff
@@ -14,27 +14,17 @@ public class CameraView : View
 
 		internal event EventHandler ShutterClicked;
 
-		public static readonly BindableProperty ShutterCommandProperty =
-			BindableProperty.Create(nameof(ShutterCommand),
-						typeof(ICommand),
-						typeof(CameraView));
+		internal static readonly BindablePropertyKey ShutterCommandPropertyKey =
+			BindableProperty.CreateReadOnly(nameof(ShutterCommand),
+				typeof(ICommand),
+				typeof(CameraView),
+				default,
+				BindingMode.OneWayToSource,
+				defaultValueCreator: ShutterCommandValueCreator);
 
-		public ICommand ShutterCommand
-		{
-			get => (ICommand)GetValue(ShutterCommandProperty);
-			set => SetValue(ShutterCommandProperty, value);
-		}
-
-		public static readonly BindableProperty ShutterCommandParameterProperty =
-			BindableProperty.Create(nameof(ShutterCommandParameter),
-						typeof(object),
-						typeof(CameraView));
+		public static readonly BindableProperty ShutterCommandProperty = ShutterCommandPropertyKey.BindableProperty;
```

Comment:
How about getting rid of ShutterCommandPropertyKey? (we can just put ".BindableProperty" at the end of the chain)

Reply:
Is there anything wrong about:
`
public static readonly BindableProperty ShutterCommandProperty = BindableProperty.Create(nameof(ShutterCommand), typeof(ICommand), typeof(CameraView), defaultBindingMode: BindingMode.OneWayToSource);
`
Just curious what's the difference to the BindablePropertyKey approach.

Reply:
When I was reading the docs and Xamarin.Forms platform code I noticed this is a common place patter for ReadOnly bindable properties. I had the same question the other night when I was reviewing this code and came to the conclusion that I think this is okay.

As far as I understand the `BindablePropertyKey` allows for readonly bindable properties. 

If we use that traditional approach, I think there will be problems. Given the following scenario
- Using BindableProperty.Create
- Using getter/setter
- Using `BindingMode.OneWayToSource`

The code would allow the ViewModel to set the `ICommand` property but it won't be updated on control. This means the ViewModel will lose it's `ICommand` reference to the shutter and the code won't be able to execute it from the ViewModel.

Reply:
I think the BindablePropertyKey is correct looking at the docs also.  It kind of hurts my head thinking about it, I've never had a need for a VM to use a command to execute something on a control/view, it's always View action to VM execute something.